packages/web/src/index.css

```diff
@@ -156,37 +156,37 @@
   .title {
     font-size: 3rem;
     line-height: 1;
-    font-weight: 700;
+    font-weight: 500;
```

Comment:
These styles apply all across the app. Just double checking that:
1. this is intentional
2. you've done a thorough QA of the whole app to make sure it doesn't look weird in some places

Reply:
updated, added a screenshot below